verifier/src/errors.rs

```diff
@@ -74,6 +77,13 @@ impl fmt::Display for VerifierError {
             Self::FriVerificationFailed(err) => {
                 write!(f, "verification of low-degree proof failed: {err}")
             }
+            Self::InsufficientConjecturedSecurity(minimal_security,proofs_security)=> {
+                write!(f, "verification failed due to parameters not providing enough conjectured security: expected at least {minimal_security} bits of security but the proof provides only {proofs_security} bits of conjectured security")
+            }
+            Self::InsufficientProvenSecurity(minimal_security,proofs_security)=> {
+                write!(f, "verification failed due to parameters not providing enough proven security: expected at least {minimal_security} bits of security but the proof provides only {proofs_security} bits of proven security")
+            }
+            Self::UnacceptableProofOptions => {write!(f, "verification failed due to parameters not providing enough security: the proof was generated with a set of parameters not accepted by the verifier")}
```

Comment:
I would rephrase it a bit since this could fail not only if security is too low but also if the security level is higher or the same, but just with a different parameter set. Maybe something like: "invalid proof options: security parameters do not match the acceptable parameter set".